CMakeLists.txt

```diff
@@ -1,70 +0,0 @@
-cmake_minimum_required(VERSION 3.2)
```

Comment:
I don't think delete file is the appropirate way to remove file from the pull requests.

Reply:
It will delete the file in the main repo, when the change is accepted